sstables/storage.hh

```diff
@@ -74,6 +74,8 @@ public:
 class storage {
     friend class test;
 
+    std::filesystem::path _base_dir;    // Local base directory (of table)
```

Comment:
Too late, maybe, but this field doesn't make sense for s3 backend. It's "location" is not location, but a key in the system.sstables, which should be schema ID, not `/var/lib/.../keyspace/table-uuid` string. And after this patch it's abused even more :(

Reply:
Looks like people are digging the whole you are trying to fill.

I think you should send a PR to make yourself the owner of storage.*, so you get notification when people want to change it.

Reply:
Ah, I see Benny did request a review from you.